### ISSUE TITLE
Flashing Improvements

### ISSUE THREAD

**Issue by nicinabox** [nicinabox/lets-split-guide#8]
I've noticed folks having trouble with the flashing instructions as well as a fair amount of misinformation. I'd like to expand the Flashing section to include:

- Parts from QMK's LS [readme on Flashing](https://github.com/qmk/qmk_firmware/tree/master/keyboards/lets_split#flashing)
- What is EE_HANDS for?
- When is eeprom flashing needed?
- Do both sides have to be connected to flash? (no)
- Do I have to flash both sides? (yes)
- Flashing on ARM-based machines like RPi 
- ...other things?

**Comment by nicinabox** [comment]
avrdude on raspi 

 ```
echo "deb http://apt.adafruit.com/raspbian/ jessie main" | sudo tee --append /etc/apt/sources.list
wget -O - -q https://apt.adafruit.com/apt.adafruit.com.gpg.key | sudo apt-key add -
sudo apt-get update
sudo apt-get install avrdude dfu-programmer gcc-avr avr-libc
```

**Comment by nicinabox** [comment]
Addressed in #9
